dev/migrate.ps1

```diff
@@ -2,20 +2,21 @@
 # Creates the vault_dev database, and runs all the migrations.
 
 # Due to azure-edge-sql not containing the mssql-tools on ARM, we manually use
-#  the mssql-tools container which runs under x86_64. We should monitor this
-#  in the future and investigate if we can migrate back.
-# docker-compose --profile mssql exec mssql bash /mnt/helpers/run_migrations.sh @args
+#  the mssql-tools container which runs under x86_64.
 
 param(
-  [switch]$all = $false,
-  [switch]$postgres = $false,
-  [switch]$mysql = $false,
-  [switch]$mssql = $false,
-  [switch]$sqlite = $false,
-  [switch]$selfhost = $false,
-  [switch]$pipeline = $false
```

Comment:
This is actually not valid syntax and is also not necessary. Switch parameters default to false if not present.